packages/gamut-labs/src/brand/LandingPageSection/LandingPageFeature.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import { Heading, Markdown } from '@codecademy/gamut';
+
+const Icon = styled.img`
+  width: 4rem;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
+`;
+
+const Image = styled.img`
+  width: 100%;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
+`;
+
+const StyledMarkdown = styled(Markdown)`
+  font-size: 1rem;
+`;
+
+export type LandingPageFeatureProps = {
+  /**
+    Used to determine if an icon or a full size image should be rendered
+  */
+  isIcon?: boolean;
+  imgSrc: string;
+  /**
+   Hero image alt text (for screen readers)
+  */
+  imgAlt: string;
+  title?: string;
+  /**
+    Main body text (can include html)
+  */
+  desc?: string;
+  /**
+    Hero image URL
+  */
+  testId?: string;
+};
+
+export const LandingPageFeature: React.FC<LandingPageFeatureProps> = ({
+  isIcon = false,
```

Comment:
Nit: it's falsy by default 😄 

```suggestion
  isIcon,
```